src/components/reusuables/sections/YohdevSection.js

```diff
@@ -0,0 +1,33 @@
+import React from "react"
+import { Container, Row, Col } from "react-bootstrap"
+import YohDevButton from "../../YohDevButton"
+import SectionHeader from "../typography/SectionHeader"
+
+const YohdevSection = (props) => {
```

Comment:
Change props to the prop name. Example, (props)-> ({heading,subtitle})

Reply:
props changed to prop name